modules/acquia_cms_common/composer.json

```diff
@@ -55,7 +55,8 @@
         "drupal/acquia_cms_starter": "<1.3.3",
         "drupal/acquia_cms_toolbar": "<1.4",
         "drupal/acquia_cms_tour": "<1.3.2 || ~2.0.0",
-        "drupal/acquia_cms_video": "<1.4"
+        "drupal/acquia_cms_video": "<1.4",
+        "drupal/acquia_connector": "<4.0.6"
```

Comment:
@rajeshreeputra Looks like this is is no longer required as Acquia CMS no longer uses/override any connector service.

Reply:
Yes Acquia CMS Common no longer uses any connector service but it is required on acquia server to connect.